### PR TITLE
Unit test for shredder

### PR DESCRIPTION
**Pull request by amolgautam25**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -4,6 +4,9 @@ go 1.13
 
 require (
 	github.com/aws/aws-sdk-go v1.31.13
+	github.com/golang/mock v1.4.3
+	github.com/onsi/ginkgo v1.11.0
```
Comment (iamkirkbater) [inline go.mod:8]: You shouldn't need these two deps, `ginkgo` and `gomega` because you don't seem to be using BDD or the gomega matching.

Comment (amolgautam25) [reply go.mod]: removed the `ginko` , `gomega` dependency

File: pkg/awsManager/aws_manager_test.go [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+func TestDeleteEc2Instance(t *testing.T) {
+	testCases := []struct {
+		title           string
+		setupAWSMock    func(r *mock.MockClientMockRecorder)
+		listOfInstances []*string
+		errorExpected   bool
+	}{
+		{
+			title: "test 1 - No buckets passed",
```
Comment (fahlmant) [inline pkg/awsManager/aws_manager_test.go:85]: buckets -> instances

File: pkg/awsManager/aws_manager_test.go [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+			errorExpected:   false,
+		}, {
+			title: "test 2 - Invalid Instances passed",
+			setupAWSMock: func(r *mock.MockClientMockRecorder) {
+				r.TerminateInstances(gomock.Any()).Return(&ec2.TerminateInstancesOutput{}, errors.New("Error")).AnyTimes()
+			},
+			listOfInstances: []*string{aws.String("abcd"), aws.String("abcd")},
+			errorExpected:   true,
+		}, {
+			title: "test 3 - valid Buckets passed",
```
Comment (fahlmant) [inline pkg/awsManager/aws_manager_test.go:99]: buckets -> instances

File: pkg/awsManager/aws_manager_test.go (hunk 2 above)
Comment (amolgautam25) [reply pkg/awsManager/aws_manager_test.go]: done

File: pkg/awsManager/aws_manager_test.go (hunk 3 above)
Comment (amolgautam25) [reply pkg/awsManager/aws_manager_test.go]: done

**Comment by fahlmant** [comment]
Looks good :+1: Please squash

**Pull request merged**